### DISCUSSION TITLE
Very different readings between detailed and not

### DISCUSSION THREAD

**Discussion by clara-j** [jertel/vuegraf#163] [General]
I noticed something strange with one of Vue CTs and the reported values.  I noticed that my graphs had strange jumps from 100 to 700 over and over at some times and then would be around 100 for a while.  I looked into it and notice when I looked at the same data in detailed view (to see if there was some quick spikes) it was much higher the whole time with no spikes.  I checked the other CTs and they all appear to roughly match (detailed vs false), so seems unique to this one.

Now this CT does have multiple Emporia outlets nested under it, so maybe its some edge case related to that.  

When I look in the Emporia app I see the CT reporting around 100, with the items under it 300.  So I think the total is about 400-500 on average.  When I click on the CT I see the 400-500.

<img width="1369" alt="Screenshot 2024-08-30 at 4 12 27 PM" src="https://github.com/user-attachments/assets/288c14d1-aa2a-4816-b7de-56b4973af58b">

**Comment by jertel** [comment]
Have you looked at the raw values? Perhaps there's some pattern that you can identify. I've not personally seen this behavior but I think there was a discussion on here a while ago that talked about how Emporia handles nested devices. I don't have any nested devices with my configuration so I am unlikely to be able to reproduct what you're seeing.
